core/client/views/settings/navigation.js

```diff
@@ -1,5 +1,12 @@
 import BaseView from 'ghost/views/settings/content-base';
 
-var SettingsNavigationView = BaseView.extend();
+var SettingsNavigationView = BaseView.extend({
+    keyDown: function (event) {
+        if (event.keyCode === 187) {
```

Comment:
Man, I thought `=` was a weird button to add stuff, but your
>Yep.

seemed so solid that I had no more doubts :D